### ISSUE TITLE
GRS80 ellipsoid is used instead of WGS84

### ISSUE THREAD

**Issue by AdamRzepka** [gadomski/utm#6]
Hi, the documentation says, that library uses WGS84 model. However, it looks like the ellipsoid parameters are actually taken from GRS80. Please compare:
 - [WGS84](https://epsg.io/4326)
 - [GRS80](https://epsg.io/42310)

Code:
```rust
const WGS84: Ellipsoid = Ellipsoid {
    a: 6378137.0,
    f: 1.0 / 298.257222101,
};
```

I can provide PR with an actual WGS84 ellipsoid, if it's ok for you.

**Comment by gadomski** [comment]
Yup, PR welcome! Would be cool to support both to but not necessary :-). Thanks for the catch.
